### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by yhaggai**

### CODE REVIEW

File: the-then-years-bug/the-ten-years-bug.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+Developers often make mistakes. Bugs go unnoticed, uncaught by unit tests, overlooked by QA and reach production. Sometimes it's a just a minor issue, sometimes it's major bug that costs you loose data and users' confidence. Whatever it is, the issue gets fixed. You patch the bug, mend corrupt data and everything gets back to it's place.
 
-Surly there isn't a bug that is unfixable right?
+Surely there isn't a bug that is unfixable right?
 
 Our snippet
 ------------
-As a prologue to this story I'll tell you about our snippet. To use [Jaco](https://getjaco.com), just sign up, copy a small snippet to your site, and your users sessions starts to get recorded. The main goal of the snippet is this:
+As a prologue to this story I'll tell you about our snippet. To use [Jaco] - (https://getjaco.com), just sign up, copy a small snippet to your site, and your users sessions start to be recorded. The main goal of the snippet is this:
```
Comment (alexpusch) [inline the-then-years-bug/the-ten-years-bug.md:10]: the - is unnecessary, [text](url) is markdown for a link

Comment (alexpusch) [inline the-then-years-bug/the-ten-years-bug.md:10]: I'm not sure if its "start to get recorded" or "start to be recorded". Both sound weird...

update: I've asked Micky and he says that it should be "your users' sessions start to be recorded"

File: the-then-years-bug/the-ten-years-bug.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ We simply create a new script tag on the hosting page, which loads our main reco
 
 First symptoms
 ------------
-It all started with a collection of unrelated bugs. An 'unable to get property '2' of undefined' here, an 'undefined is not a function' there. All seemingly minor bugs. But for some reason those became legendary. We fixed them, but they persisted. We could not longer recreate them, but they kept popping up in the logs consistently and regularly. Many hours were spent in the attempt to debug them.
+It all started with a collection of unrelated bugs. An 'unable to get property '2' of undefined' here, an 'undefined is not a function' there. All seemingly minor bugs. But in some reason those became legendary. We fixed them, but they persisted. We could not recreate them, but they kept popping up in the logs consistently and regularly. Many hours were spent in the attempt to debug them.
```
Comment (alexpusch) [inline the-then-years-bug/the-ten-years-bug.md:22]: Why in? pretty sure it's "for some reason"

I think the "longer" should stay there. But it should be "no longer" instead of "not longer"

Comment (yhaggai) [reply the-then-years-bug/the-ten-years-bug.md:22]: you're right 'in' shouldn't be there
I would just drop the word ( not 'in' nor 'for')

File: the-then-years-bug/the-ten-years-bug.md [hunk 3]
```diff
@@ -69,18 +69,18 @@ Now lets look at the header we attached to our recorder.js file:
 'Cache-Control': 'max-age=315360000, no-transform, public'
 ```
 
-In other words we just told the all of our end users browsers to cache our recorder for 10 years without downloading new versions, or doing any kind of communication with S3 to check for a change in the caching policy. All the minor bugs we fixed in the past still linger in our end users caches, and worse, we don't have anything to do about it. Fuck.
+We just told the all of our end users browsers to cache our recorder for 10 years without downloading new versions, or doing any kind of communication with S3 to check for a change in the caching policy. All the minor bugs we fixed in the past still linger in our end users caches, and worse, we don't have anything to do about it. Fuck.
```
Comment (alexpusch) [inline the-then-years-bug/the-ten-years-bug.md:72]: lol the first sentence is a mess. "the all of our" :)

should be "we just told all of our end users..."

File: the-then-years-bug/the-ten-years-bug.md [hunk 4]
```diff
@@ -69,18 +69,18 @@ Now lets look at the header we attached to our recorder.js file:
...
 
-Well it couldn't be helped. We've created recorder_v2.js, published an email with an explanation and instruction how to change the snippet. As an auxiliary fix we've configured S3 to permanent redirect all request to recorder.js to recorder_v2.js
+Well it couldn't be helped. We've created recorder_v2.js, and sent out an email to all our customers with an explanation and instructions how to change the snippet. As an auxiliary fix we've configured S3 to permanently redirect all requests to recorder.js to recorder_v2.js
 
 These actions decreased the amount of old recorder.js file being used, but not eliminated it completely. Some clients are less cooperative with request such as ours and did not updated their snippet. Theoretically some bugs will hunt us for another 10 years.
 
 In conclusion
 ------------
-We're not sure how this have come to pass. But we've all learned from it and be much more careful when configuring cacheing policy.
+We're not sure how this has come to pass. But we've all learned from it and be will much more careful when configuring cacheing policy.
```
Comment (alexpusch) [inline the-then-years-bug/the-ten-years-bug.md:84]: "will be" instead of "be will"

**Comment by alexpusch** [comment]
👍  Thanks!

File: the-then-years-bug/the-ten-years-bug.md (hunk 4 above)
Comment (yhaggai) [reply the-then-years-bug/the-ten-years-bug.md:84]: bah. my bad

**Comment by alexpusch** [comment]
I've manually merged this PR to my branch. If you have more comment put them in #2
